packages/lesswrong/themes/defaultPalette.ts

```diff
@@ -291,8 +292,7 @@ export const defaultComponentPalette = (shades: ThemeShadePalette): ThemeCompone
     diffInserted: "#d4ead4",
     diffDeleted: "#f0d3d3",
     usersListItem: shades.greyAlpha(.05),
-    primaryDim: '#d3edf2',
-    primaryDim2: '#e2f1f4',
+    primaryDim: '#e2f1f4',
```

Comment:
Turns out `background.primaryDim` wasn't being used anyway, so I removed it and renamed `primaryDim2`